`test_ytsaurus_select_subset_of_columns` is flaky

Example failure from #85920 [here](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=85920&sha=392c6124eecfe5ae3b86a4c9f2f82f28ad972854&name_0=PR)  and #87994 [here](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=87994&sha=640b7026bf68b1e1aeb3615e36ad0bf6abb96493&name_0=PR):

```
File: test_ytsaurus/test_tables.py:475 - in test_ytsaurus_select_subset_of_columns
    yt = YTsaurusCLI(started_cluster, instance, YT_HOST, YT_PORT)
E   NameError: name 'YT_HOST' is not defined
```

--- a/tests/integration/compose/docker_compose_ytsaurus.yml
+++ b/tests/integration/compose/docker_compose_ytsaurus.yml
@@ -10,4 +10,4 @@ services:
             YT_PROXY: ytsaurus_backend1:${YTSAURUS_PROXY_PORT}
         ports:
             - ${YTSAURUS_PROXY_PORT}:${YTSAURUS_PROXY_PORT}
-        entrypoint: yt_local start --proxy-port ${YTSAURUS_PROXY_PORT} --local-cypress-dir /var/lib/yt/local-cypress --ytserver-all-path /usr/bin/ytserver-all --sync --fqdn ytsaurus_backend1 --proxy-config '{coordinator={public_fqdn="ytsaurus_backend1:${YTSAURUS_PROXY_PORT}"}}' --node-count 1 --queue-agent-count 1 --address-resolver-config '{enable_ipv4=%true;enable_ipv6=%false;}' --native-client-supported --id primary -c '{name=query-tracker}'  --enable-auth --create-admin-user
+        entrypoint: yt_local start --proxy-port ${YTSAURUS_PROXY_PORT} --local-cypress-dir /var/lib/yt/local-cypress --ytserver-all-path /usr/bin/ytserver-all --sync --fqdn ytsaurus_backend1 --proxy-config '{coordinator={public_fqdn="ytsaurus_backend1:${YTSAURUS_PROXY_PORT}"}}' --node-count 1 --queue-agent-count 1 --address-resolver-config '{enable_ipv4=%true;enable_ipv6=%false;}' --port-range-start 50000 --native-client-supported --id primary -c '{name=query-tracker}'  --enable-auth --create-admin-user

